lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -36,27 +30,21 @@ magma::SessionSet create_session_set_req(
   uint32_t sess_ver_no                         = info.ver_no;
   magma::SessionState::SessionInfo::NodeId tmp = info.nodeId;
   std::string node_id                          = tmp.node_id;
-  uint64_t f_seid_                             = info.Seid.f_seid;
-  req.set_seid(seid);
-  req.set_sess_ver_no(sess_ver_no);
+  req.set_subscriber_id(seid);
```

Comment:
why is subscriber id set as session id?

Reply:
In proto it is changed to subscribed id instead of imsi, so the name changed to.